lab-10-larry/lib/list.js

```diff
@@ -0,0 +1,41 @@
+'use strict';
+
+const Node = require(__dirname + '/node');
+
+const LinkedList = module.exports = function(node) {
+  this.head = node || null;
+};
+
+LinkedList.prototype.prepend = function(value) {
+  let oldHead = this.head;
+  this.head = new Node(value, oldHead)
+}
+
+LinkedList.prototype.append = function(value) {
+  if(this.head === null) return this.head = new Node(value);
+  let current = this.head;
+  while(current.next !== null)
```

Comment:
Really need to use your {} here ... and indent the working line below.  That's a pattern you want to be setting now, so that as you get into coding in a larger company, you don't run into trouble.